plasmapy/formulary/tests/test_densities.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for functionality contained in `plasmapy.formulary.frequencies`."""
+import astropy.units as u
+import numpy as np
+import pytest
+
+from plasmapy.formulary.densities import plasma_critical_density
+from plasmapy.formulary.frequencies import plasma_frequency
+
+n_i = 5e19 * u.m**-3
+ion = "e-"
+
+
+def test_plasma_critical_density():
+    r"""Test the plasma_critical_density function in frequencies.py."""
+
+    omega_p = plasma_frequency(n=n_i, particle=ion)
+    n_c = plasma_critical_density(omega_p)
+
+    assert n_c.unit.is_equivalent(u.m**-3)
```

Comment:
These two tests are good! However you should split them into two separate test functions, so that if the first fails the other still gets tested. If you want to calculate n_c once for both tests, look into pytest Fixtures